packs/gatsby-build/config.yml

```diff
@@ -0,0 +1,29 @@
+
+# Name of the pack
+name: Gatsby Build
+
+# Description of the pack
+description: |+
+  The template pack allows you to get visibilility into the build time of your Gatsby Site,
+  using Open Telemetry to collect each step as Trace Data
+
+# Support level: New Relic | Verified | Community
+level: New Relic
+
+# Design
+icon: icon.jpeg
+logo: logo.png
+
+# Authors of the pack
+authors:
+  - New Relic
+  - Ruairi Douglas
+
+references:
```

Comment:
We should remove the references as this is not relevant for this pack, right?

Reply:
ye I had actually removed these but hadn't pushed it, they're not relevant for this instance